src/utils/sourcemaps.rs

```diff
@@ -548,8 +548,28 @@ impl SourceMapProcessor {
             }
         };
 
-        // We don't need the RAM bundle sourcemap itself
-        self.sources.remove(&sourcemap_url);
+        // We have to include the bundle sourcemap which is the first section
+        // in the bundle source map before the modules maps
+        if let Some(sourcemap_source) = self.sources.get(&sourcemap_url) {
```

Comment:
You need to check the presence of this file earlier anyway. I would suggest replacing the block starting at line 532 with
```rust
        let Some(sourcemap_source) = self.sources.get(&sourcemap_url) else {
            warn!(
                "Cannot find the sourcemap for the RAM bundle using the URL: {}, skipping",
                sourcemap_url
            );
            return Ok(());
        };
        let sourcemap_content = &sourcemap_source.contents;
```
and removing this check.